common/export.js

```diff
@@ -0,0 +1,53 @@
+(function () {
+    'use strict';
+
+    angular.module('chaise.export', [])
+
+    .directive('export', ['$http', function ($http) {
+        return {
+            restrict: 'AE',
+            templateUrl: '../common/templates/export.html',
+            scope: {
+                reference: "<",
+                hasValues: "<"
+            },
+            link: function (scope, element, attributes) {
+                scope.isLoading = false;
+
+                scope.submit = function () {
```

Comment:
Submit may not be the right name for this function but it doesn't matter right now.